CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # Changelog
 
+## [0.21.0] - 2023-09-13
+**Added**
+- (Input) Implement extended controller button support (paddles, touchpad, misc button)
+- (Input) Implement client controller metadata support
+- (Input/Windows) Add support for Applications (context menu) key
+- (Input/Windows) Implement touch, motion, and battery for DS4 input
+- (Input) Advertise support for new input features to clients
+- (Linux/Debian) Added Debian Bookworm package
+- (Prep-Commands) Expose connection environment variables
+- (Input) Implement pen and touch support for Windows
+- (Capture/Windows) Add standalone NVENC encoder
+- (Capture) Implement AV1 encoding
+- (Network) Implement IPv6 support
+- (Capture/Windows) Add option to disable realtime hags
+
+**Changed**
+- (Input) Reduce latency by implementing input batching
+- (Logging) Move input packet debug prints off the control stream thread
+- (Input) Refactor gamepad emulation code to sue DS4 extended report format
+- (Graphics/NVIDIA) Decrease GPU scheduling priority
```

Comment:
"Add an option to decrease GPU scheduling priority to workaround HAGS video hang" (the change to make it default was reverted)

Reply:
```suggestion
```